system-test/datastore.ts

```diff
@@ -868,6 +868,10 @@ describe('Datastore', () => {
       assert.strictEqual(entities.length, characters.length);
     });
 
+    it('should construct filters by null status', async () => {
+      datastore.createQuery('Character').filter('status', null);
+      datastore.createQuery('Character').filter('status', '=', null);
```

Comment:
It would be better to assert the returning result of this filter operation in all tests.